.travis.yml

```diff
@@ -50,6 +49,17 @@ jobs:
         - kill %1
       after_success:
         - authelia-scripts docker push-image --arch=$ARCH
+        - if [[ $TRAVIS_TAG ]]; then docker create --name authelia-binary clems4ever/authelia:$TRAVIS_TAG-$ARCH && docker cp authelia-binary:/usr/app/authelia ./authelia-linux-$ARCH; fi
+      before_deploy:
+        - echo Deploying Authelia $TRAVIS_TAG-$ARCH to GitHub Releases
+      deploy:
+        provider: releases
+        api_key: '$GITHUB_API_KEY'
```

Comment:
I will create the API key and set it in Travis.

Reply:
Done

Reply:
Great.